crypto/src/main/kotlin/web5/sdk/crypto/KeyManager.kt

```diff
@@ -46,11 +45,21 @@ public interface KeyManager {
    * Signs the provided payload using the private key identified by the provided alias.
    *
    * @param keyAlias The alias referencing the stored private key.
-   * @param payload The data/payload to be signed.
+   * @param signingInput The data to be signed.
+   * @return The signature in JWS R+S format
    *
    * Implementations should ensure that the signing process is secured, utilizing secure cryptographic
    * practices and safeguarding the private key during the operation. The specific signing algorithm
    * used may depend on the type and parameters of the stored key.
    */
-  public fun sign(keyAlias: String, payload: Payload)
+  public fun sign(keyAlias: String, signingInput: ByteArray): ByteArray
+
+  /**
+   * Return the alias of [publicKey], as was originally returned by [generatePrivateKey].
+   *
+   * @param publicKey A public key in JWK (JSON Web Key) format
+   * @return The alias belonging to [publicKey]
+   * @throws IllegalArgumentException if the key is not known to the [KeyManager]
+   */
+  public fun getDefaultAlias(publicKey: JWK): String
```

Comment:
Given that `JWK` already have a `kid` field, is it possible to use that field to identify keys?

IMO, it's easier to put the alias within that field, rather than expanding the interface.

Reply:
Yeah, this a good point, and kicked off a big discussion. Both AwsKeyManager and InMemoryKeyManager are setting the alias into `kid`, so it would work with the code we have here.
 
It's not clear that the publicKeyJwk will always have a `kid` though. It's optional. I think it's up to the DID method how that publicKeyJwk is put together?

Reply:
I'm gonna merge this now to unblock #63, but happy to keep the convo going in Slack